src/main/java/com/microsoft/serviceClient/GraphServiceClient.java

```diff
@@ -3,19 +3,21 @@
 import com.azure.core.credential.TokenCredential;
 import com.microsoft.graph.requests.BaseGraphRequestAdapter;
 import com.microsoft.graph.requests.GraphClientOption;
+import com.microsoft.graph.requests.IBaseClient;
 import com.microsoft.info.Constants;
 import com.microsoft.kiota.RequestAdapter;
 import com.microsoft.kiota.authentication.AnonymousAuthenticationProvider;
 import com.microsoft.kiota.authentication.AuthenticationProvider;
 import com.microsoft.kiota.authentication.AzureIdentityAuthenticationProvider;
 
 import okhttp3.OkHttpClient;
+import org.jetbrains.annotations.NotNull;
```

Comment:
Why do we use `org.jetbrains.annotations.NotNull`?

Reply:
ah, good catch, fixed